### PR TITLE
Ignore network errors

### PR DESCRIPTION
**Pull request by tung2744**
ref DEV-2274

### CODE REVIEW

File: authui/src/errors.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// Modified from https://github.com/sindresorhus/is-network-error/blob/main/index.js
```
Comment (louischan-oursky) [inline authui/src/errors.ts:1]: I see the original work has an extra checking for Safari 17+ to check if the error has an undefined stack. I guess we need to port that as well.

**Review by louischan-oursky — CHANGES_REQUESTED** [review]

File: authui/src/errors.ts (hunk 1 above)
Comment (tung2744) [reply authui/src/errors.ts:1]: In my testing that check incorrectly filter out network errors in safari (stack is not undefined), so I removed that check.

Comment (louischan-oursky) [reply authui/src/errors.ts:1]: Fair. Can you include this test as comment to explain this is one of the modification? It helps us to understand why the code is different from the original work.

Comment (louischan-oursky) [reply authui/src/errors.ts:1]: Let me do this for you.

**Review by louischan-oursky — APPROVED** [review]

**Pull request merged**